### PR TITLE
[JENKINS-67515] Fix vertical icon alignment for build status in M and S icon sizes

### PR DESCRIPTION
**Pull request by janfaracik**
See [JENKINS-67515](https://issues.jenkins-ci.org/browse/JENKINS-67515).

**Before**
image

**After**
image

### Proposed changelog entries

* Fix vertical icon alignment for build status in medium (M) and small (S) icon sizes.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@daniel-beck 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [x] There are at least 2 approvals for the pull request and no outstanding requests for change
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [x] Proper changelog labels are set so that the changelog can be generated automatically
- [x] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [x] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Comment by janfaracik** [comment]
Just spotted an issue with the status icon in titles (eg on a build page) - fixing it now

**Review by timja — APPROVED** [review]
nice

**Review by NotMyFault — APPROVED** [review]
Works like a charm 👏🏻 

I closed my issue in favor of Daniel's to track the progress there.

**Comment by daniel-beck** [comment]
> I closed my issue in favor of Daniel's

I didn't see yours, sorry!

**Review by daniel-beck — APPROVED** [review]
If it works 🤷 (untested)

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by oleg-nenashev — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Thanks for the patch @janfaracik !

File: war/src/main/less/modules/icons.less
Comment (daeho-ro) [inline war/src/main/less/modules/icons.less:97]: This line break the build history UI in the left bottom for each job, see

image

Comment (janfaracik) [reply war/src/main/less/modules/icons.less:97]: Heya, what browser/OS are you using? Thanks!

Comment (daeho-ro) [reply war/src/main/less/modules/icons.less:97]: I am using Chrome in Desktop with version 97.0.4692.71, thanks for reply :)

Comment (NotMyFault) [reply war/src/main/less/modules/icons.less:97]: Weird enough I can replicate that on my pipelines, but other kinds of job types, like freestyle or maven, are unaffected.

<details>
<summary>Pipeline Job</summary>



</details>

<details>
<summary>Freestyle job</summary>



</details>

Comment (timja) [reply war/src/main/less/modules/icons.less:97]: Lots of stuff is duplicated in pipeline from core which has caused issues like this before

Comment (timja) [reply war/src/main/less/modules/icons.less:97]: Caused by Pipeline stage view plugin modifying a core class =/

image

Comment (timja) [reply war/src/main/less/modules/icons.less:97]: https://github.com/jenkinsci/pipeline-stage-view-plugin/pull/193

File: war/src/main/less/base/layout-commons.less
Comment (timja) [inline war/src/main/less/base/layout-commons.less:128]: Either this or the build status icon change has caused the progress bar to move to the left during builds in the console log and on the run page

image

**Comment by basil** [comment]
Caused [JENKINS-68356](https://issues.jenkins.io/browse/JENKINS-68356).

**Pull request merged**
